hibernate-core/src/test/java/org/hibernate/orm/test/jpa/query/NativeQueryOrdinalParametersTest.java

```diff
@@ -18,106 +18,114 @@
 
 	@Test
 	@TestForIssue(jiraKey = "HHH-10885")
-	public void testNativeQueryOrdinalParameter() {
-		doInJPA( this::entityManagerFactory, entityManager -> {
-			Query query = entityManager.createNativeQuery( "SELECT * FROM GAME g WHERE title = ?" );
-			query.setParameter( 1, "Super Mario Brothers" );
-			List list = query.getResultList();
-			assertEquals( 1, list.size() );
-		} );
+	public void testNativeQueryOrdinalParameter(EntityManagerFactoryScope scope) {
+		scope.inTransaction(
+				entityManager -> {
+					Query query = entityManager.createNativeQuery( "SELECT * FROM GAME g WHERE title = ?" );
+					query.setParameter( 1, "Super Mario Brothers" );
+					List list = query.getResultList();
+					assertEquals( 1, list.size() );
+				}
+		);
 	}
 
 	@Test
 	@TestForIssue(jiraKey = "HHH-11121")
-	public void testConflictWithSessionNativeQuery() {
+	public void testConflictWithSessionNativeQuery(EntityManagerFactoryScope scope) {
 		final String sqlString = "SELECT * FROM GAME g WHERE title = ?";
 
-		doInJPA( this::entityManagerFactory, entityManager -> {
-			NativeQuery sqlQuery = entityManager.unwrap( Session.class ).createNativeQuery( sqlString );
-			sqlQuery.setParameter( 1, "Super Mario Brothers" ).setCacheable( true );
+		scope.inTransaction(
+				entityManager -> {
+					NativeQuery sqlQuery = entityManager.unwrap( Session.class ).createNativeQuery( sqlString );
+					sqlQuery.setParameter( 1, "Super Mario Brothers" ).setCacheable( true );
 
-			List results = sqlQuery.list();
-			assertEquals( 1, results.size() );
+					List results = sqlQuery.list();
+					assertEquals( 1, results.size() );
 
-			NativeQuery query = (NativeQuery) entityManager.createNativeQuery( sqlString );
-			query.setParameter( 1, "Super Mario Brothers" );
-			List list = query.list();
-			assertEquals( 1, list.size() );
+					NativeQuery query = (NativeQuery) entityManager.createNativeQuery( sqlString );
+					query.setParameter( 1, "Super Mario Brothers" );
+					List list = query.list();
+					assertEquals( 1, list.size() );
 
-			sqlQuery = entityManager.unwrap( Session.class ).createNativeQuery( sqlString );
-			sqlQuery.setParameter( 1, "Super Mario Brothers" ).setCacheable( true );
+					sqlQuery = entityManager.unwrap( Session.class ).createNativeQuery( sqlString );
+					sqlQuery.setParameter( 1, "Super Mario Brothers" ).setCacheable( true );
 
-			results = sqlQuery.list();
-			assertEquals( 1, results.size() );
+					results = sqlQuery.list();
+					assertEquals( 1, results.size() );
 
-			query.setParameter( 1, "Super Mario Brothers" );
-		} );
+					query.setParameter( 1, "Super Mario Brothers" );
+				}
+		);
 	}
 
 	@Test
 	@TestForIssue(jiraKey = "HHH-12532")
 	// Add RequiresDialect be Cockroach version 201
-	@RequiresDialect({ PostgreSQL82Dialect.class})
-	public void testCteNativeQueryOrdinalParameter() {
+	@RequiresDialect( value = PostgreSQLDialect.class, version = 820 )
+	@RequiresDialect( value = CockroachDialect.class, version = 820 )
```

Comment:
I think this should be this way:
```suggestion
	@RequiresDialect( value = CockroachDialect.class, version = 2010 )
```